balancer-js/src/modules/exits/exits.module.ts

```diff
@@ -441,7 +521,14 @@ export class Exit {
       outputReferences,
     });
 
-    return call;
+    let tokenOutAmount = Relayer.isChainedReference(limits[0])
```

Comment:
The way `createBatchSwap` is currently implemented, I don't think limits will ever be chained references. I tried using it, but the transactions were failing because the limits were above `MaxInt256` if I'm not mistaken. That's why it's hardcoded to that max value.

Reply:
This is a good point. We couldn't actually use chained ref values in the Relayer if we wanted to. For safety reasons that Relayer won't allow limits to be set that way. I'll remove those checks as there's no need for them.